runtime/ztests/expr/exists.yaml

```
@@ -0,0 +1,17 @@
+script: |
+  super -s -c 'values exists(select * from (values ("foo"), ("bar")))'
+  echo // ===
+  super -s -c 'values exists(values null)'
+  echo // ===
+  super -s -c 'values exists(values "foo", "bar" | where this == "baz")'
```

Nit: Maybe make this a SPQ-style test?

```suggestion
spq: |
  values
    exists(select * from (values ("foo"), ("bar"))),
    "==="
    exists(values null),
    "===",
    exists(values "foo", "bar" | where this == "baz")
```